Helm lint fails: dependencies in Chart.yaml with apiVersion v1; jobflow file:// path incorrect

### Description

When linting the Helm chart under `installer/helm/chart/volcano`, `helm lint` reports that `dependencies` are not allowed in `Chart.yaml` when `apiVersion` is `v1`. Separately, the local subchart reference for `jobflow` used `file://../charts/jobflow`, which does not match the repository layout: the `jobflow` chart lives under `installer/helm/chart/volcano/charts/jobflow`, so the path should be relative to the volcano chart root (e.g. `file://charts/jobflow`).

This breaks CI or local validation that runs `helm lint` on the chart, and can confuse anyone running `helm dependency update` with an invalid `file://` URL.

### Steps to reproduce the issue

1. Clone the Volcano repository and `cd` to the repo root (or use your fork/branch that still has `dependencies` inside `Chart.yaml` with `apiVersion: v1`, and the old `file://../charts/jobflow` URL if reproducing the original state).
2. Run:
   ```bash
   helm lint installer/helm/chart/volcano
   ```
3. Optionally run:
   ```bash
   helm dependency update installer/helm/chart/volcano
   ```
   and verify whether the `jobflow` dependency resolves from the intended directory.

### Describe the results you received and expected

**Received:**  
`helm lint` fails with an error similar to:

```text
[ERROR] Chart.yaml: dependencies are not valid in the Chart file with apiVersion 'v1'. They are valid in apiVersion 'v2'
```

With the incorrect `file://` path, dependency resolution does not point at `volcano/charts/jobflow` as intended.

**Expected:**  
`helm lint` passes without requiring `apiVersion: v2` if the project chooses to keep Helm v1–style `Chart.yaml`. Dependencies should be declared in a way compatible with that choice (e.g. `requirements.yaml` for v1 charts), and the `file://` URL should resolve to `installer/helm/chart/volcano/charts/jobflow`.


### What version of Volcano are you using?

1.14

### Any other relevant information

_No response_

--- a/installer/helm/chart/volcano/Chart.yaml
+++ b/installer/helm/chart/volcano/Chart.yaml
@@ -1,7 +1,7 @@
-apiVersion: v1
+apiVersion: v2
 description: A Helm chart for Volcano
 name: volcano
-version: "1.5"
+version: "1.5.0"
 appVersion: "0.1"
 icon: https://raw.githubusercontent.com/volcano-sh/charts/master/docs/images/volcano-logo.png
 home: https://volcano.sh
@@ -10,7 +10,7 @@ sources:
 dependencies:
 - name: jobflow
   version: "1.0.0"
-  repository: "file://../charts/jobflow"
+  repository: "file://charts/jobflow"
 maintainers:
   - name: k82cn
     email: klaus1982.cn@gmail.com

--- a/installer/helm/chart/volcano/Chart.yaml
+++ b/installer/helm/chart/volcano/Chart.yaml
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A Helm chart for Volcano
 name: volcano
-version: "1.5.0"
+version: "1.15.0"
 appVersion: "0.1"
 icon: https://raw.githubusercontent.com/volcano-sh/charts/master/docs/images/volcano-logo.png
 home: https://volcano.sh
@@ -10,7 +10,7 @@ sources:
 dependencies:
 - name: jobflow
   version: "1.0.0"
-  repository: "file://charts/jobflow"
+  repository: "file://./charts/jobflow"
 maintainers:
   - name: k82cn
     email: klaus1982.cn@gmail.com

--- a/installer/helm/chart/volcano/charts/jobflow/Chart.yaml
+++ b/installer/helm/chart/volcano/charts/jobflow/Chart.yaml
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
 appVersion: "0.1"
 description: A Helm chart for Volcano-Job-flow
 name: jobflow

